test/test_twobytwo_indexed.jl

```diff
@@ -498,5 +507,351 @@ solve!(m)
+set_fixed!(PF[:l], true)
+solve!(m)
+# PF.l=1
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[l]†Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["x.L","PF.l=1"]#  56.5057343
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[k]†Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["x.K","PF.l=1"]#  44.2432973
+@test MPSGE.Complementarity.result_value(m._jump_model[:Y][:x]) ≈ TwoxTwo_DemandIndPrice_Nest["y.x","PF.l=1"]#  1.1301147
+@test MPSGE.Complementarity.result_value(m._jump_model[:Y][:y]) ≈ TwoxTwo_DemandIndPrice_Nest["y.y","PF.l=1"]#  1.1028032
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[l]†Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["y.L","PF.l=1"]#  23.1620512
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[k]†Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["y.K","PF.l=1"]#  27.2034033
+@test MPSGE.Complementarity.result_value(m._jump_model[:U]) ≈ TwoxTwo_DemandIndPrice_Nest["U._","PF.l=1"]#  1.1209365
+@test MPSGE.Complementarity.result_value(m._jump_model[:C][:ra]) ≈ TwoxTwo_DemandIndPrice_Nest["RA._","PF.l=1"]#  179.1727363
+@test MPSGE.Complementarity.result_value(m._jump_model[:PC][:x]) ≈ TwoxTwo_DemandIndPrice_Nest["PC.x","PF.l=1"]#  1.1301147
+@test MPSGE.Complementarity.result_value(m._jump_model[:PC][:y]) ≈ TwoxTwo_DemandIndPrice_Nest["PC.y","PF.l=1"]#  1.1581026
+@test MPSGE.Complementarity.result_value(m._jump_model[:PF][:l]) ≈ TwoxTwo_DemandIndPrice_Nest["PF.L","PF.l=1"]#  1
+@test MPSGE.Complementarity.result_value(m._jump_model[:PF][:k]) ≈ TwoxTwo_DemandIndPrice_Nest["PF.K","PF.l=1"]#  1.2771592
+@test MPSGE.Complementarity.result_value(m._jump_model[:PU][:l]) ≈ TwoxTwo_DemandIndPrice_Nest["PU.L","PF.l=1"]#  1.278736
+@test MPSGE.Complementarity.result_value(m._jump_model[:PU][:k]) ≈ TwoxTwo_DemandIndPrice_Nest["PU.K","PF.l=1"]#  1
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[x]†U")]) ≈ TwoxTwo_DemandIndPrice_Nest["DU.x","PF.l=1"]#  100.8187947
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[y]†U")]) ≈ TwoxTwo_DemandIndPrice_Nest["DU.y","PF.l=1"]#  49.191153
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU[l]‡U")]) ≈ TwoxTwo_DemandIndPrice_Nest["SU.L","PF.l=1"]#  75
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU[k]‡U")]) ≈ TwoxTwo_DemandIndPrice_Nest["SU.K","PF.l=1"]#  75
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[x]‡Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["SY.x","PF.l=1"]#  100
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[y]‡Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["SY.y","PF.l=1"]#  50
+# For the counterfacturals where there two nested final demands are equal, summing matches our results, but for this they are different and summing the two doesn't match
+# @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC→CUρC[ra]")]) ≈ TwoxTwo_DemandIndPrice_Nest["FDRA.L","PF.l=1"]+TwoxTwo_DemandIndPrice_Nest["FDRA.K","PF.l=1"]#  84.0702388 +  71.6690945
```

Comment:
Following from comment above: Here the result for the variable we report doesn't equal the sum of the 2 from GAMS.